lib/helpers.js

```diff
@@ -241,6 +241,32 @@ function getDeployableItemsNumber(functions) {
   return _.sum([_.keys(functions).length].concat(_.map(functions, f => _.size(f.events))));
 }
 
+function setExponentialInterval(cb, initialDelay, maxDelay) {
+  let delay = initialDelay;
+  let timer;
+  const internalCb = () => {
```

Comment:
The naming `cb` `internalCb` is not that readable, I would use something more meaningful like `targetFunction` and `timerWrapper`.

Also, `initialDelay` doesn't seem like a very useful parameter now (it was more important before to calculate the total delay). Can we remove it?

Reply:
I agree: `targetFunction` and `timerWrapper` are a lot better than `cb` and `internalCb` 👍
About initialDelay (and maxDelay) parameters: the setExponentialInterval could be a multipurpose util function. This is the reason I added the two parameters. It's something like the original setInterval but with an exponential behaviour.
Anyway, If you prefer I can remove the two parameters ;)

Reply:
yes, I didn't mean to delete the two parameters to control the interval, just the one for the initialDelay since it's less useful than `maxDelay`. In any case, having both doesn't hurt.